storage-bigtable/src/access_token.rs

```diff
@@ -105,13 +109,20 @@ impl AccessToken {
                 .compare_and_swap(false, true, Ordering::Relaxed)
             {
                 // Refresh already pending
+                let token_refresh_time = self.token_refresh_start_time.load(Ordering::SeqCst);
```

Comment:
Any reason why you didn't use `Ordering::Relaxed` here?

Reply:
Also I'd prefer if `token_refresh_time` was renamed to `token_refresh_start_time` for (arguable) improved code clarity